### PR TITLE
Depth Chart

### PR DESCRIPTION
**Pull request by Renzenar**
I added the position data for PA and wOBA and have the dropdown switch working. I plan to add a second function for the pitching values so that the batting and pitching positions can be changed separately, which will make future UI much easier and be better for a user.

### CODE REVIEW

File: Team_Roster/app/routes.py [hunk 1]
```diff
@@ -19,4 +19,4 @@ def roster(teamId):
 @app.route('/<teamId>/depthChart', methods=['GET'])
 def depthChart(teamId):
     year = request.args.get("year")
-    return ShowDepthChart(teamId, year)
\ No newline at end of file
+    return ShowDepthChart(teamId)
```
Comment (Jarpletz) [inline Team_Roster/app/routes.py]: why not pass in the year anymore, don't you need it to calculate the depth chart???

File: Team_Roster/pages/depthChart.py [hunk 2]
```diff
@@ -1,44 +1,99 @@
...
 from sqlalchemy import func
 
-def ShowDepthChart(teamId, year):
-    #Query to get player fielding time in each position for the specified team and year
-    subquery = (
-        db.session.query(
-            Fielding.position,
-            Fielding.playerID,
-            func.sum(Fielding.f_InnOuts).label("total_innouts"),
+def ShowDepthChart(teamId):
```
Comment (Jarpletz) [inline Team_Roster/pages/depthChart.py]: are you fetching all the stats, or just the one displayed?

File: Team_Roster/app/templates/depthChart.html [hunk 3]
```diff
@@ -1,14 +1,37 @@
...
-{% for position, players in position_data.items() %}
+<!-- Dropdown to change stat -->
+<form method="GET" action="/{{ teamID }}/depthChart">
+    <label for="stat">Select Stat: </label>
+    <select name="stat" id="stat" onchange="this.form.submit()">
+        <option value="percentage" {% if stat == 'percentage' %}selected{% endif %}>Time Percentage</option>
+        <option value="PA" {% if stat == 'PA' %}selected{% endif %}>Plate Appearances</option>
+        <option value="wOBA" {% if stat == 'wOBA' %}selected{% endif %}>wOBA</option>
+    </select>
+    <input type="hidden" name="year" value="{{ year }}">
```
Comment (Jarpletz) [inline Team_Roster/app/templates/depthChart.html:14]: nice!

**Review by Jarpletz — COMMENTED** [review]
Had one question about how you're processing GET data differently from everywhere else in the app, you may want to consider extracting it fully inside `routes.py` to be consistent with else where. Otherwise, looks great, and is functionally very smooth!

File: Team_Roster/app/routes.py (hunk 1 above)
Comment (Renzenar) [reply Team_Roster/app/routes.py]: I was getting it from the URL, but I went ahead and changed it

File: Team_Roster/pages/depthChart.py (hunk 2 above)
Comment (Renzenar) [reply Team_Roster/pages/depthChart.py]: I'm not quite sure what you mean, but currently only the ones actually displayed are being retrieved at the moment.

**Review by Jarpletz — APPROVED** [review]

**Pull request merged**